### PR TITLE
Allow forwarding duply logs to logger command

### PR DESCRIPTION
**Pull request by mcarrolle**
By default, duply command forward stdout to a file.
I add a way to forward output to logger command. The logger command write messages to system logs (syslog, journald...).

The default behaviour is to use "file" output for duply logs

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -51,8 +51,15 @@
 # [*duply_archive_executable*]
 #   Set the symbolic path pointing to the configured duply executable when installing the archive from sourceforge.
 #
-# [*duply_version*]
```
Comment (tohuwabohu) [inline manifests/init.pp:54]: nit: did this get lost on purpose?

File: manifests/init.pp [hunk 2]
```diff
@@ -131,6 +138,8 @@
   Stdlib::Absolutepath $duply_archive_install_dir = $duplicity::params::duply_archive_install_dir,
   Optional[String] $duply_version = undef,
   Stdlib::Absolutepath $duply_archive_executable = $duplicity::params::duply_archive_executable,
+  String $duply_log_output = $duplicity::params::duply_log_output,
```
Comment (tohuwabohu) [inline manifests/init.pp:141]: Given this parameter requires additional logic inside the module, this is a great fit for an `Enum`, what do you think?

```suggestion
  Enum['file', 'logger'] $duply_log_output = $duplicity::params::duply_log_output,
```

File: spec/classes/init_spec.rb [hunk 3]
```diff
@@ -148,5 +148,13 @@
       )
     }
   end
+
+  describe 'with duply_log_output => logger' do
```
Comment (tohuwabohu) [inline spec/classes/init_spec.rb:152]: :+1:

File: spec/defines/profile_spec.rb [hunk 4]
```diff
@@ -7,7 +7,8 @@
       duplicity::public_key { 'key1': content => 'key1' }
       duplicity::public_key { 'key2': content => 'key2' }
 
-      class { 'duplicity': }
```
Comment (tohuwabohu) [inline spec/defines/profile_spec.rb:10]: nit: looks like an accidental change?

File: spec/defines/profile_spec.rb [hunk 5]
```diff
@@ -395,4 +396,42 @@ class { 'duplicity': }
       )
     end
   end
-end
+
+  describe 'with duply_log_output equals logger and cron_enabled' do
```
Comment (tohuwabohu) [inline spec/defines/profile_spec.rb:400]: :smiley: awesome, regression tests, love it!

**Review by tohuwabohu — CHANGES_REQUESTED** [review]

**Comment by tohuwabohu** [comment]
Awesome, looking good, thanks a lot! Please have a look at the suggestion. Thanks!

File: manifests/init.pp (hunk 1 above)
Comment (mcarrolle) [reply manifests/init.pp:54]: Oh, yes sorry.

File: manifests/init.pp (hunk 2 above)
Comment (mcarrolle) [reply manifests/init.pp:141]: Yes, for sure, it's better to use Enum here.

File: spec/defines/profile_spec.rb (hunk 4 above)
Comment (mcarrolle) [reply spec/defines/profile_spec.rb:10]: Yes, Fixed

**Review by mcarrolle — COMMENTED** [review]
Suggested changes has been pushed.
Thank you for the time allocated to the maintenance of this project

**Comment by tohuwabohu** [comment]
Released to the forge as part of 6.3.0, thanks a lot!

**Pull request merged**